dtc/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Andy Gross <andy.gross@linaro.org>
+
+pkgname=dtc
+pkgver=1.4.4
```

Comment:
I see.  Well i just worked up a patch to make this equivalent to the master dtc branch.  I'll go ahead and push that so this will work in all cases.  When the new version of dtc is tagged we can drop the patch.